### ISSUE TITLE
Character Pass does not increase progress

### ISSUE THREAD

**Issue by A2198382hhhh** [itsamewayu/OLFallenDoll#2075]
There is a bug. The progress of the pass does not increase, but the research progress does. A solution is needed.
![c9f97fb8-b97f-49cb-9302-50a0274c3ebe](https://github.com/user-attachments/assets/a2a6bc73-a381-4afe-bd63-4b74d2be7471)
![93713aa3-87c1-460e-a556-653a8162e3c6](https://github.com/user-attachments/assets/0714a2b8-c7ff-4b31-bc56-9541cd80297c)
![9e47906b-fc60-4483-838e-9572347aed66](https://github.com/user-attachments/assets/806865c1-8c33-4068-b732-aa6250442269)
![9bf24743-84de-4b04-a10f-a66ca145dea4](https://github.com/user-attachments/assets/3ae08e7a-6412-44ac-8be9-d29f79333654)
![cc088a1d-c476-4e82-b51e-843e3ac3321b](https://github.com/user-attachments/assets/260cd3df-e65a-484b-97ca-13470a1e3e91)

**Comment by hardcoretj** [comment]
i got this 2 after buying some tiers had 200 left to max out the pass and got no progress

**Comment by Kuckich** [comment]
I tried switching to a character with level 0 and zero progress, raised one level with the help of missions and does not rise anymore.

**Comment by hardcoretj** [comment]
So it’s still bugged?

**Comment by hardcoretj** [comment]
yeah i had 200 points left to max it out so i did and no progress after  buying some tiers

**Comment by Kuckich** [comment]
yes

**Comment by hardcoretj** [comment]
maybe the weekly reset will fix it
